src/scss/components/_navbar.scss

```diff
@@ -0,0 +1,64 @@
+//------------------------------------------------------------------------------
+// @component: Navbar
+//------------------------------------------------------------------------------
+// @description
+//
+//------------------------------------------------------------------------------
+
+//
+// @settings
+
+// Layout Variables
+
+
+//
+// @scss
+.c-navbar {
+	overflow: hidden;
+	background-color: #f1f1f1;
+
+	&-menu {
+		list-style-type: none;
+		margin: 0;
+		padding: 0;
+	}
+
+	&-menu-link {
+		float: left;
+		font-size: 17px;
```

Comment:
pls use material design specs... all font-sizes should be based on material design and all units should be based on 8px grid